frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts

```diff
@@ -210,7 +214,7 @@ export class CastDragAndDrop implements OnInit {
       };
       this.castPositions.push(castPosition);
       for (let dancerIndex = 0; dancerIndex < castPosition.dancerCount;
-          dancerIndex++) {
+           dancerIndex++) {
         castPosition.castRows.push({
           subCastDancers: new Array(this.castCount),
```

Comment:
Is the extra space in front of dancerIndex++) { a typo or purposeful?

Reply:
Looks like it's lining up with the open parenthesis of the for-loop